developers/weaviate/concepts/indexing/vector-indexes.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Vector indexes overview
+sidebar_position: 10
+image: og/docs/indexing.jpg
+# tags: ['vector index plugins']
+---
+
+Weaviate is a vector database. Most objects in Weaviate collections have one or more vectors. Individual vectors can have thousands of dimensions. Collections can have millions of objects. The resulting vector space can be exceedingly large.
+
+Weaviate uses vector indexes to [efficiently search](https://weaviate.io/blog/why-is-vector-search-so-fast) the vector space. Different vector index types offer trade-offs in resource use, speed, and accuracy.
+
+import VectorIntro from '/_includes/indexes/vector-intro.mdx';
+
+<VectorIntro/>
+
+## Vector indexing
+
+[Vector embeddings](https://weaviate.io/blog/vector-embeddings-explained) are arrays of elements that can capture meaning. The original data can come from text, images, videos, or other content types. A model transforms the underlying data into an embedding. The elements in the embedding are called dimensions. High dimension vectors, with thousands of elements, capture more information, but they are harder to work with.
+
+Vector databases make it easier to work with high dimensional vector embeddings. Embeddings that capture similar meanings are closer to each other than embeddings that capture different meanings. To find objects that have similar semantics, vector databases must efficiently calculate the "distance" between the objects' embeddings.
```

Comment:
I think this could be improved by explicitly mentioning the problem of "search". Why does it need to compare meaning, and why at speed?

Reply:
Adding to this - the biggest reason for having a vector index is to scale vector search. So I think the page should set out the need to have performant, scalable search, and how an index solves it.